shell/platform/android/io/flutter/view/FlutterView.java

```diff
@@ -592,7 +592,11 @@ public final WindowInsets onApplyWindowInsets(WindowInsets insets) {
             mMetrics.systemGestureInsetBottom = systemGestureInsets.bottom;
             mMetrics.systemGestureInsetLeft = systemGestureInsets.left;
         }
-        updateViewportMetrics();
```

Comment:
These sections are to ensure we never have a 0-sized viewport

Reply:
Is this related to the performance regression?

Reply:
Sort-of.  See my comment in viewport_metrics.cc

This allows the benchmarks and examples to succeed on Android, while still keeping the checks in viewport_metrics.cc.

Your discretion as to which ones I keep or remove.  If we keep the checks in viewport_metrics.cc, its possible iOS and Windows embedder will need a similar change to this.

On another note, I'm very surprised CI tests didn't catch this one....

Reply:
Reverted these changes since we use >= in viewport_metrics.cc